src/changeColors.ts

```diff
@@ -0,0 +1,15 @@
+import * as vscode from 'vscode';
+import { getSetting } from './getColor';
+
+type ColorCustomization = { [key: string]: string | undefined }
+
+export default async (color?: string) => {
+    const settings = vscode.workspace.getConfiguration('workbench');
+    const currentColorCustomization: ColorCustomization = settings.get('colorCustomizations') || {};
+    const tabBarBorderColor = (getSetting('tabBorder') || true) ? color: undefined
+    const titleBarBackgroundColor = getSetting('titleBackground') ? color : undefined
```

Comment:
`const titleBarBackgroundColor = getSetting('titleBackground') ? color : undefined`
I would change to:

`const titleBarBackgroundColor = getSetting().titleBackground && color`

Reply:
I think ternary is needed as otherwise it would get type of `const tabBarBorderColor: string | false | undefined`